### DISCUSSION TITLE
Data from custom blade layouts are not in $request

### DISCUSSION THREAD

**Discussion by ValerianMemsk** [orchidsoftware/platform#2409] [Q&A] [answered]
I have A LOT of specific access permissions in my app (~60) and utilizing them with basic "checkbox-dump" is quite uncomfortable
I tried to reorganize them into custom table like this:
```
<body>
  <form name='permissions'>
  <table>
    <thead>
      <tr>
        <td>{{ __('Permission') }}</td><td>{{ __('Status') }}</td><td class='hide'></td>
      </tr>
    </thead>
    <tbody>
      @foreach($fields as $field)
        <tr>
          <td>{{ $field['description'] }}</td>
          <td><input type='checkbox' @if($field['active']) checked @endif></td>
          <td style='display:none'>{{ $field['slug'] }}</td>
        </tr>
      @endforeach
    </tbody>
  </table>
  </form>
</body>
```
Displaying all of them is a-ok, but when i try to save changes made in this table - nothing is happening. Variable `$request` send to `save(Request $request)` method containing every other "block" of data, but not this one. 
I wasnt able to find solution in other discussions, so can anyone help me? How can i put my data into request?

**Comment by tabuna** [comment, marked as answer]
In your example, I see the following problems:

1. the screen already contains a form, it doesn't need to be declared again.
2. Each input must have a unique name that will be sent to the server

**Reply by ValerianMemsk** [reply]
I appreciate the help, good sir.
Adding "name" attribute solved my problem.

**Comment by luizfilho0205** [comment]
@tabuna , 
To group the data into an array, we set the 'name' attribute to `array[key]`; In this way, the request data will be in an associative array and we will be able to access its values by key. Is this correct?
For example, for inputs

```
<input name="visa[debit]" value="0.75">
<input name="visa[credit1x]" value="2.30">

```

The request data will be:
```
   "visa" => [
     "debit" => "0.75"
     "credit1x" => "2.30"
   ]
```
But what if I need to add one more level to the nesting? Setting the 'name' attribute to something like `owner[array[key]]` doesn't seem to work.
How should the 'name' attribute be set so that the request data looks like:
```
"foo" => [
   "visa" => [
     "debit" => "0.75"
     "credit1x" => "2.30"
   ]
],
"bar" => [
   "visa" => [
     "debit" => "0.95"
     "credit1x" => "1.30"
   ]
]
```
How to do this correctly?
Thank's for your time.
